sdk/tests/conformance2/uniforms/gl-uniform-arrays-sub-source.html

```diff
@@ -258,11 +258,19 @@
 
 ];
 
-function checkUniformValues(program, typeInfo) {
+function checkUniformValues(program, typeInfo, initialValues) {
   var numSrcValues = typeInfo.srcValues.length / typeInfo.numElementsPerType;
-  for (var ii = 0; ii < numSrcValues; ++ii) {
-    var expectedValues = typeInfo.srcValues.slice(
-        typeInfo.numElementsPerType * ii, typeInfo.numElementsPerType * (ii + 1));
+  for (var ii = 0; ii < 3; ++ii) {
```

Comment:
It's the fragment shader's uniform |color| array size. I used a var with comment.